docs/api/en/objects/Sprite.html

```diff
@@ -78,8 +78,8 @@ <h3>[method:Sprite copy]( [param:Sprite sprite] )</h3>
 
 		<h3>[method:null raycast]( [param:Raycaster raycaster], [param:Array intersects] )</h3>
 		<p>
-		Get intersections between a casted ray and this sprite.
-		[page:Raycaster.intersectObject] will call this method.
+		Get intersections between a casted ray and this sprite. [page:Raycaster.intersectObject]() will call this method.
+		Notice that you have to setup your raycaster with [page:Raycaster.setFromCamera]() when raycasting against sprites.
```

Comment:
>The raycaster must be initialized by calling [page:Raycaster.setFromCamera]() before raycasting against sprites.

---
"Notice" is not the correct word in this case, usually it would be Note, or Warning, or Alert, Hint. But neither are needed in this case, IMO.

Reply:
Done!

Reply:
I like my suggested wording better. "Setup" is a noun.